packages/api/endpoints/executions.js

```diff
@@ -79,6 +85,32 @@ async function del(req, res) {
   return res.send({ message: 'Record deleted' });
 }
 
+/**
+ * Get execution history for a single granule or multiple granules
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function history(req, res) {
+  const { granuleId } = req.params;
+  const payload = req.body;
+  const granuleIds = granuleId ? [granuleId] : await getGranuleIdsForPayload(payload);
+
+  const knex = await getKnexClient();
+
+  const executionPgModel = new ExecutionPgModel();
+
+  const executionCumulusIds = await getGranuleExecutionCumulusIds(knex, ['granule_id'], granuleIds);
+
+  const executions = await executionPgModel
+    .getExecutionsFromCumulusIds(knex, executionCumulusIds);
+
+  return res.send(executions);
+}
+
+router.post('/history', validateBulkGranulesRequest, history);
+router.get('/history/:granuleId', history);
```

Comment:
What is the rationale for having a GET and a POST endpoint to both return executions by granule IDs? Why not just always use the POST that can handle both?

Reply:
Yeah the original thinking was to allow for an easier single granule request, but probably isn't necessary to have both. Removed the GET